src/Microsoft.Android.Sdk.ILLink/PreserveLists/Mono.Android.xml

```diff
@@ -36,15 +35,7 @@
 		-->
 		<type fullname="Java.Interop.TypeManager*JavaTypeManager" />
 		<type fullname="Java.Lang.Object">
-			<field name="handle" />
-			<field name="handle_type" />
-			<field name="refs_added" />
 			<method name="SetHandleOnDeserialized" />
 		</type>
-		<type fullname="Java.Lang.Throwable">
-			<field name="handle" />
-			<field name="handle_type" />
-			<field name="refs_added" />
-		</type>
```

Comment:
It looks ok to remove these, we have this:
https://github.com/dotnet/android/blob/7aed4e3f2d3c9d76790520e176e7ce3b2c093b41/src/Microsoft.Android.Sdk.ILLink/PreserveLists/Java.Interop.xml#L3-L14

Reply:
Commit  130905e1612f1c3f41b6b233056cae2753270630would have not passed unit tests if `PreserveLists/Java.Interop.xml` did not exist.